gulpfile.js

```diff
@@ -1,67 +1,66 @@
-var debug = true;
-
+var autoprefixer = require('gulp-autoprefixer');
+var babelify = require('babelify');
+var browserify = require('browserify');
+var buffer = require('vinyl-buffer');
+var cache = require('gulp-cache');
+var del = require('del');
+var eslint = require('gulp-eslint');
 var gulp = require('gulp');
 var gutil = require('gulp-util');
-var sass = require('gulp-sass');
 var imagemin = require('gulp-imagemin');
-var sass = require('gulp-sass');
-var del = require('del');
-var runSequence = require('run-sequence');
+var minifycss = require('gulp-minify-css');
+var notify = require('gulp-notify');
 var rename = require('gulp-rename');
-var browserify = require('browserify');
-var babelify = require('babelify');
+var runSequence = require('run-sequence');
+var sass = require('gulp-sass');
 var source = require('vinyl-source-stream');
-var buffer = require('vinyl-buffer');
 var sourcemaps = require('gulp-sourcemaps');
-var notify = require('gulp-notify');
-var eslint = require('gulp-eslint');
-var autoprefixer = require('gulp-autoprefixer');
-var minifycss = require('gulp-minify-css');
 var uglify = require('gulp-uglify');
-var cache = require('gulp-cache');
+
+var IS_DEBUG = true;
 
 // Lint the gulpfile
-gulp.task('selfie', function(){
+gulp.task('selfie', function selfieTask() {
```

Comment:
How does this affect anonymous arrow functions in ES6 / babel? We'll probably want to use a lot of those.